docs/tutorials/AddRemark.adoc

```diff
@@ -38,7 +38,7 @@ We accomplish that by returning a `CommandResult` with an accompanying message.
 ----
 package seedu.address.logic.commands;
 
-import seedu.address.model.Model;
+import Model;
```

Comment:
Don't need to change this file since its not part of our project. But import Model wouldn't actually work if this was code haha!